tests/test-extensions/petstore-extension/src/main/java/org/mule/test/petstore/extension/PetStoreOperations.java

```diff
@@ -129,4 +139,17 @@ public void setSecureCage(@Optional @NullSafe List<String> providers, String use
   }
 
   public void makePhoneCall(PhoneNumber phoneNumber) {}
+
+  public static class CorrelationInfoOutputResolver implements OutputTypeResolver<CorrelationInfo> {
+
+    @Override
+    public MetadataType getOutputType(MetadataContext context, CorrelationInfo key) {
+      return context.getTypeLoader().load(CorrelationInfo.class);
```

Comment:
what are you testing here?

Reply:
nothing. Because I'm returning an interface, SDK forces me to put a custom resolver